zeppelin-web/pom.xml

```diff
@@ -120,34 +152,77 @@
           </execution>
 
           <execution>
-            <id>yarn install</id>
-            <goals>
-              <goal>yarn</goal>
-            </goals>
-            <configuration>
-              <arguments>install --no-lockfile</arguments>
-            </configuration>
+              <id>yarn add bower</id>
+              <goals>
+                  <goal>yarn</goal>
+              </goals>
+              <configuration>
+                <!--ignore-scripts-->
+                  <arguments>add bower --no-lockfile --no-bin-links</arguments>
```

Comment:
> My main goal is to make developers free from local env for start development and build.

I assume you meant "local env" -> "local env other than osx, linux". Because at least, osx, linux doesn't require any additional steps to build 'zeppelin-web' module except for just run 'mvn package'.

Like i mentioned above, run 'yarn add xx' changes 'package.json', which is version controlled file. And many section in 'pom.xml' is duplicated with 'package.json'.

How about find alternative way to solve the problem without touching package.json and without duplicating the feature in two different places?